realm/realm-library/src/objectServer/java/io/realm/mongodb/StreamNetworkTransportImpl.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.mongodb;
```

Comment:
Is it possible to move this class to the internal package?

Reply:
Don't think so, we need it to provide access to package protected methods on user and app. What we can do is to to package protect its constructor.

Reply:
Yes, that sounds like a good idea. If it can't be internal then we need to add docs 😓